include/tables/scene_table.h

```diff
@@ -0,0 +1,130 @@
+/**
+ * Scene Table
+ *
+ * DEFINE_SCENE should be used for most scenes
+ *    - Argument 0: Name of the scene segment in spec
+ *    - Argument 1: Enum value for this scene
+ *    - Argument 2: Title card's text id, or zero for no title card
+ *    - Argument 3: Scene draw config index
+ *    - Argument 4: Restriction flags. See `RESTRICTIONS_SET` on z_parameter.c
+ *    - Argument 5: Persistent cycle flag: switch0
+ *    - Argument 6: Persistent cycle flag: switch1
+ *    - Argument 7: Persistent cycle flag: chest
+ *    - Argument 8: Persistent cycle flag: collectible
```

Comment:
Tbh, I think just having one argument with a macro like `RESTRICTIONS_SET` would be cleaner.

Reply:
I agree, but since I don't understand those flags that well I'm not sure how it should look like.
Something like this?
```
#define PERSISTENT_FLAGS_SET(switch0, switch1, chest, collectible) ...
```

Reply:
Yeah, though perhaps add `CYCLE` i.e `PERSISTENT_CYCLE_FLAGS_SET`?